source/features/highlight-file-in-prs.tsx

```diff
@@ -0,0 +1,100 @@
+import './highlight-file-in-prs.css';
+import React from 'dom-chef';
+import select from 'select-dom';
+import cache from 'webext-storage-cache';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {getOwnerAndRepo, getRepoURL} from '../libs/utils';
+import {isSingleFile} from '../libs/page-detect';
+import getDefaultBranch from '../libs/get-default-branch';
+import {groupSiblings} from '../libs/group-buttons';
+import * as icons from '../libs/icons';
+
+async function init(): Promise<void> {
+	const {ownerName, repoName} = getOwnerAndRepo();
+	const cacheKey = `highlight-file-in-prs:${ownerName}/${repoName}`;
+
+	let files = await cache.get<Record<string, string[]>>(cacheKey);
+	if (files === undefined) {
+		files = await fetch();
+		cache.set(cacheKey, files, 3);
+	}
+
+	console.log('files', files);
+
+	const path = select('clipboard-copy, #blob-edit-path')!.getAttribute('value')!; // `clipboard-copy` on blob page, `#blob-edit-path` on edit page.
+	console.log('path', path);
+	if (!files[path]) {
+		return;
+	}
+
+	const wrapper = <div className="rgh-highlight-file-in-prs" />;
+	for (const pr of files[path]) {
```

Comment:
There is probably no room for more then 10 PR links.

```suggestion
	for (const pr of files[path].slice(0, 10)) {
```